src/bp/ui-studio/src/web/views/Content/index.tsx

```diff
@@ -82,25 +82,37 @@ class ContentView extends Component<Props, State> {
     this.props.contentItems.forEach((element: ContentElementUsage) => {
       element.usage = []
       Object.values(this.props.flows.flowsByName).forEach((flow: FlowView) => {
-        flow.nodes.forEach((node: NodeView) => {
-          const usage: ContentUsage = {
-            type: 'Flow',
-            name: flow.name,
-            node: node.name,
-            count: 0
-          }
+        // Skip skill flows
+        if (!flow.skillData) {
```

Comment:
We tend to use guard statement instead of nesting. The result it the exact same but is a bit easier to read
```ts
if(flow.skillData) {
return
}

// rest or your code goes here, not nested
```

Reply:
oh, I think this doesn't apply here